### PR TITLE
Add Note for Windows User in NewComer

### LINKED ISSUE #2845: Direct Link to Docs for Windows Users
### Description

The Installation Section on the Docs page can easily confuse a Windows User to do a lot of setup, when there is an easy way to deploy using Releases and the Documentation is very easy to understand and read for the Windows Part, but not easily accessible. 

Hence my suggestion is to add a note for Windows Users that directs them to the Release-method setup considering he is a NEW COMER. 😄 

### Link to Documentation Page
https://scancode-toolkit.readthedocs.io/en/latest/getting-started/newcomer.html#before-you-start-using-scancode
Add a note to 
https://scancode-toolkit.readthedocs.io/en/latest/getting-started/install.html#installation-on-windows-10

### Select Category

-  [ ] Inconsistency
-  [ ] New Section Request
-  [x] General Improvement
-  [ ] Typo/Mistakes
-  [x] Other

I have the PR ready already, will be linking this issue in the same as soon as I send it. 😄

### PR DESCRIPTION
**Pull request by OsmiumOP**
Signed-off-by: Harsh Bansal <harshbansal.contact@gmail.com>

Fixes #2845 
Simple addition to documentation I feel should be a good thing for any New Comer like me 😄 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

**Comment by OsmiumOP** [comment]
@pombredanne can you check this?

File: src/cluecode/copyrights.py [hunk 1]
```diff
@@ -2565,6 +2565,9 @@ def build_detection_from_node(
     #holder is Tim Hudson (tjh@mincom.oz.au).
     COPYRIGHT: {<HOLDER> <JUNK> <NAME-EMAIL>}       #83001
 
+    # Copyright lowRISC contributors.
```
Comment (pombredanne) [inline src/cluecode/copyrights.py]: How is this related to this PR?

**Review by pombredanne — CHANGES_REQUESTED** [review]
hey, there may be some mixup? There is entirely unrelated code in this PR.

File: src/cluecode/copyrights.py (hunk 1 above)
Comment (OsmiumOP) [reply src/cluecode/copyrights.py]: Should I re-fork and do the changes again?

Comment (pombredanne) [reply src/cluecode/copyrights.py]: Your call.... this can work. Also learning git merge/rebase/amend is a very useful skill to nurture.

Comment (OsmiumOP) [reply src/cluecode/copyrights.py]: Ok sir, I re-forked and reopened the PR. 
Will surely look into merge/rebase/amend as soon as I get time from exams.

**Pull request closed without merging**